src/styles/variables.scss

```diff
@@ -16,4 +16,6 @@ $toc-min: 1360px;
   --layout-toc-width: 300px;
   --layout-search-height: 4.8rem;
   --font-code: 'Monaco', monospace;
+
+  --code-theme-comment: hsl(210, 4%, 71%);
```

Comment:
Amplify UI doesn't have enough in-between colors (like neutral.30) so I added a variable for this color here which should make updating for dark mode later on easier.

Reply:
I feel like we might know someone we can talk to about getting more in-between colors added to Amplify UI 😉